### DISCUSSION TITLE
support on webdriverIO v7

### DISCUSSION THREAD

**Discussion by maksym-alavatskyi** [webdriverio-community/wdio-video-reporter#55] [General]
Hi guys, is there plans to support webdriverIO v7?

**Comment by presidenten** [comment]
Recompiled with wdio v7 and there doesnt seem to be any difference. 

Will release new version soon together with a new template project which is using wdio v7.

**Comment by Suresh8236** [comment]
Hello,

Is there any chance to integrate wdio video reporting to python pytest framework.

**Reply by christian-bromann** [reply]
No! Different languages, different frameworks. But feel free to copy the reporter code into a Python reporter.

**Comment by christian-bromann** [comment]
I think I can go ahead and close this since we have shipped v8 support now and moved on 😉 Feel free to post questions if there are struggles updating to v8. We have rewritten the whole code base and there are chances of regression. Please keep me posted if so.
